### ISSUE TITLE
[Webkit] Double values aren't stored correctly

### ISSUE THREAD

**Issue by lightsprint09** [automerge/automerge#255]
Steps to reproduce:

1. load Automerge #253 performance branch into Webkit

2 execute:
```javascript
var doc3 = Automerge.init()
var doc4 = Automerge.change(doc3, function(doc) {
    doc.double = 1589032171.1
})
doc4.double
``

doc4 is not equal ` 1589032171.1`

**Comment by ept** [comment]
Fixed in 1db5d12ae0f189ca293748834f85d76fc51eeb0a. Thanks for the bug report!

**Comment by lightsprint09** [comment]
Thanks for the quick fix 🥳
